packages/termux-tools/pkg

```diff
@@ -111,40 +122,39 @@ select_mirror() {
 
 	# Test mirror availability, remove unaccessible mirrors from list.
 	echo "Testing the available mirrors:"
-	local w total_mirror_weight=0
+	local w
```

Comment:
The `w` variable should be renamed as well, since it no longer refers to `weight`

Reply:
Vagrantfile*~~

~~

Reply:
Termux